licenses/jackson-jr-objects-2.13.2.jar/META-INF/LICENSE

```diff
@@ -0,0 +1,8 @@
+This copy of Jackson JSON processor streaming parser/generator is licensed under the
+Apache (Software) License, version 2.0 ("the License").
+See the License for details about distribution rights, and the
+specific rights regarding derivate works.
```

Comment:
@trask I couldn't find a simple way to exclude `licenses` from spellcheck

https://github.com/client9/misspell

Any thoughts?

Reply:
https://github.com/client9/misspell#how-do-you-check-an-entire-folder-recursively

> The following examples selects all `.txt` files that are *not* in the `vendor` directory:
>
> ```bash
> find . -type f -name '*.txt' | grep -v vendor/ | xargs misspell -error
> ```

I can mess around with it tomorrow